java/src/main/java/ai/rapids/cudf/DType.java

```diff
@@ -23,10 +23,10 @@
   INT16(2, 2, "short"),
   INT32(4, 3, "int"),
   INT64(8, 4, "long"),
-  // UINT8(1, 5, "byte"),
-  // UINT16(2, 6, "short"),
-  // UINT32(4, 7, "int"),
-  // UINT64(8, 8, "long"),
+  UINT8(1, 5, "byte"),
```

Comment:
I deliberately left them as these since that's the equivalent Java underlying type.  TBH I'm not really sure about the purpose of this field in `DType`.

Reply:
Upon looking into this further. cudf uses this field to disable type inference on a column in CSV e.g. We need to change this to the appropriate C type. You can look at this https://github.com/rapidsai/cudf/blob/95d7c8932f61f5e4e3a1098bcffc649380090a13/cpp/tests/io/csv_test.cu#L313 test in cudf as an example of what types to use for the unsigned type

Reply:
Thanks @razajafri, I updated the schema strings per that table.